### PR TITLE
chore: `FullyQualifiedStrictTypesFixer` must run before `OrderedInterfacesFixer`

### PR DESCRIPTION
**Pull request by kubawerlos**

### CODE REVIEW

**Review by Wirone — APPROVED** [review]

**Comment by Wirone** [comment]
Thank you @kubawerlos 🍻

File: tests/Fixtures/Integration/priority/fully_qualified_strict_types,ordered_interfaces.test [hunk 1]
```diff
@@ -0,0 +1,23 @@
+--TEST--
```
Comment (keradus) [inline tests/Fixtures/Integration/priority/fully_qualified_strict_types,ordered_interfaces.test:1]: nice one.

@Wirone , i would call it a bugfix than chore - without this change, the code requires 2 passes of fixing process, should only one - now it is only one. bugfix ;)

Comment (Wirone) [reply tests/Fixtures/Integration/priority/fully_qualified_strict_types,ordered_interfaces.test:1]: You're right, I just overlooked it 🤷‍♂️.

Comment (kubawerlos) [reply tests/Fixtures/Integration/priority/fully_qualified_strict_types,ordered_interfaces.test:1]: > without this change, the code requires 2 passes of fixing process

Not in this case. Usually, a PR of such kind is a bugfix, as it changes the priorities of the fixers, although, this one isn't technically a fix, as it does not change the priorities.

Comment (keradus) [reply tests/Fixtures/Integration/priority/fully_qualified_strict_types,ordered_interfaces.test:1]: good insight.
`test` then :D sooo many types

**Pull request merged**